webapp/src/plugin.tsx

```diff
@@ -4,6 +4,7 @@
 import {Store, Action} from 'redux';
 import {PluginRegistry} from 'mattermost-webapp/plugins/registry';
 
+import ChannelHeaderMenuAction from 'components/channel_header_actions';
```

Comment:
0/5 Seems like it should be called `action` since there's only one

```suggestion
import ChannelHeaderMenuAction from 'components/channel_header_action';
```

Reply:
I thought about this also, but we could technically have more than one action in the future.

Reply:
We would have to restructure the folder probably anyhow. It would likely be 
- components/channel_header_actions
	- subscribe_header_action
		- index.ts
		- subscribe_header_action.tsx
	- other_header_action
		- index.ts
		- other_header_action

If we are going the route of planning ahead for more actions, I recommend the structure above. Otherwise, [YAGNI](https://www.martinfowler.com/bliki/Yagni.html) and it should be made singular 1/5

Reply:
renamed to singular action.